src/tracker-component.js

```diff
@@ -330,7 +336,11 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
 
             return trackCartEvent(config, 'setCart', newCart);
         },
-        removeFromCart: (data : RemoveCartData) => trackCartEvent(config, 'removeFromCart', data),
+        removeFromCart: (data : RemoveCartData) => {
+            composeCart('remove', data);
+
+            trackCartEvent(config, 'removeFromCart', data);
```

Comment:
Please give this line a look & feedback.

You'll notice that 'data' here is different from the calls to 'add' & 'set':
- 'add' & 'set' return the _entire_ cart in it's new state. This means if the cart had one item in it, and another was added in a call to 'add', the cart event will be called with two items.
- 'remove' works differently. The data will reflect only the items that were removed.

Reply:
What data did `removeFromCart` return before you made this change? I think as long as we maintain parity, then we should be fine

Reply:
It's the same as it was. Data is passed from the event directly to the tracking event.